cosmos/helpers.py

```diff
@@ -0,0 +1,13 @@
+import importlib
```

Comment:
@pankajkoti could we rename this module to `utils/importer.py`?

Also, it may be worth to log a ticket so we refactor other parts of the code that could leverage this, WDYT?

Reply:
Thanks Tatiana for the suggestion. I have created a follow-up ticket to reuse this method in other places of our existing codebase.